src/plugins/kibana/server/routes/api/ingest/register_simulate.js

```diff
@@ -18,13 +19,16 @@ export function registerSimulate(server) {
       const simulateApiDocument = request.payload;
       const body = ingestSimulateApiKibanaToEsConverter(simulateApiDocument);
 
+      const handleResponse = processESIngestSimulateResponse;
```

Comment:
If you just want to rename this function, you can use the `import as` feature `import { processESIngestSimulateResponse as handleResponse } from '../../../lib/process_es_ingest_simulate_response';`